database/src/schema/permission_group.rs

```diff
@@ -110,3 +141,24 @@ impl Statement {
         .map(|_| ())
     }
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
On a general note, I don't think we should do a `mod` for tests, this also pollutes the test-output.
What do you think?

Reply:
I thought I was following [the recommended setup](https://doc.rust-lang.org/rust-by-example/testing/unit_testing.html)?

The benefits of the mod, from my viewpoint:

* If you only use an import in `#[cfg(test)]`, you must have a separate using statement. If this is outside of the method, you end up having some pretty gnarly headers in your file due to having to try to understand what is included and what isn't. If you move the usings inside of the test, then we potentially repeat usings across multiple tests.
* You only have to write `#[cfg(test)]` once

I also can say I've never even thought about it when looking at the test output.

Reply:
- Agreed
- `#[test]` implies `#[cfg(test)]`
- When you have a bunch of tests failing, it helps, generally not really worth the tradeoff if there is any

I guess it's situational, personally I skip it unless I have extra `use` statements that I only need for `cfg(test)`.

Reply:
As per Discord I'm going to revisit these thoughts while doing changes to #17.